Futura/Futura/Signal/Signal.swift

```diff
@@ -141,7 +141,18 @@ public extension Signal {
         return self
     }
     
-    #warning("TODO: add tokens handler - either value or error without reference")
+    /// Handler used to observe all tokens passed through this Signal instance.
+    ///
+    /// - Parameter observer: Handler called every time Signal gets any token.
+    /// - Returns: Same Signal instance for eventual further chaining.
+    @discardableResult
+    func tokens(_ observer: @escaping () -> Void) -> Signal {
```

Comment:
It might be kind of hard to understand when looking at usage. Since this name has been used for some time it has been used here as well. However, if we can form a better name it might change yet.